include/asterisk/logger.h

```diff
@@ -45,6 +45,12 @@ extern "C" {
 #define VERBOSE_PREFIX_2 "  == "
 #define VERBOSE_PREFIX_3 "    -- "
 #define VERBOSE_PREFIX_4 "       > "
+#define VERBOSE_PREFIX_5 "        >> "
+#define VERBOSE_PREFIX_6 "         > "
+#define VERBOSE_PREFIX_7 "         >> "
+#define VERBOSE_PREFIX_8 "          > "
+#define VERBOSE_PREFIX_9 "          >> "
+#define VERBOSE_PREFIX_10 "          > "
```

Comment:
Why `>>` on the odd levels and `>` on the even levels?

Reply:
No particular reason, really, that was just one method of furthering the hierarchy that came to mind, you can see they progressively get more indented. Open to other suggestions, but I didn't want to add a whole bunch of different symbols either.

Reply:
> you can see they progressively get more indented

Ah, no they don't.  8 and 10 are the same. :)

I'd just make them all `>` with the progressive spaces.

Reply:
> I might be missing something, but 10 is definitely indented one more space than 8 is, if you compare with the column alignment for 9.

It only looks that way because "10" is longer than "8". If you line up the opening quotes, you can see it...

```
#define VERBOSE_PREFIX_4  "       > "
#define VERBOSE_PREFIX_5  "        >> "
#define VERBOSE_PREFIX_6  "         > "
#define VERBOSE_PREFIX_7  "         >> "
#define VERBOSE_PREFIX_8  "          > "
#define VERBOSE_PREFIX_9  "          >> "
#define VERBOSE_PREFIX_10 "          > "
```
> The downside to making them all just a single '>` more progressively spaced is that doubles the indentation as you go down. On small consoles, that can result in a lot more wasted space. The way it is now saves a few columns.

Yeah but we only have 1 level 10 verbose message and none at all for levels 6-9 so I don't see that as an issue.

If you think it really is, how about...

```
#define VERBOSE_PREFIX_4  "       > "
#define VERBOSE_PREFIX_5  "      +> "
#define VERBOSE_PREFIX_6  "     ++> "
#define VERBOSE_PREFIX_7  "    +++> "
#define VERBOSE_PREFIX_8  "   ++++> "
#define VERBOSE_PREFIX_9  "  +++++> "
#define VERBOSE_PREFIX_10 " ++++++> "
```

Reply:
> > I might be missing something, but 10 is definitely indented one more space than 8 is, if you compare with the column alignment for 9.
> 
> It only looks that way because "10" is longer than "8". If you line up the opening quotes, you can see it...

Ah, I missed that!

> ```
> #define VERBOSE_PREFIX_4  "       > "
> #define VERBOSE_PREFIX_5  "        >> "
> #define VERBOSE_PREFIX_6  "         > "
> #define VERBOSE_PREFIX_7  "         >> "
> #define VERBOSE_PREFIX_8  "          > "
> #define VERBOSE_PREFIX_9  "          >> "
> #define VERBOSE_PREFIX_10 "          > "
> ```
> 
> > The downside to making them all just a single '>` more progressively spaced is that doubles the indentation as you go down. On small consoles, that can result in a lot more wasted space. The way it is now saves a few columns.
> 
> Yeah but we only have 1 level 10 verbose message and none at all for levels 6-9 so I don't see that as an issue.

Fair enough... I think I use 6 or 7 in some of my stuff but not extensively so minor issue.

> If you think it really is, how about...
> 
> ```
> #define VERBOSE_PREFIX_4  "       > "
> #define VERBOSE_PREFIX_5  "      +> "
> #define VERBOSE_PREFIX_6  "     ++> "
> #define VERBOSE_PREFIX_7  "    +++> "
> #define VERBOSE_PREFIX_8  "   ++++> "
> #define VERBOSE_PREFIX_9  "  +++++> "
> #define VERBOSE_PREFIX_10 " ++++++> "
> ```

This is the *opposite* of what we should do since it looks like 10 is more "important" than 4... the indentation needs to be consistent on each side.

I'll just go with your original approach here, it works well enough I think.